packages/components/image/src/index.vue

```diff
@@ -246,18 +246,33 @@ export default defineComponent({
       _lazyLoadHandler = null
     }
 
+    function _wheelHandler(e){
+      if (e.ctrlKey) {
+        if (e.deltaY < 0) {
+          e.preventDefault()
+          return false
+        }
+        if (e.deltaY > 0) {
+          e.preventDefault()
+          return false
+        }
+      }
+    }
+
     function clickHandler() {
       // don't show viewer when preview is false
       if (!preview.value) {
         return
       }
+      document.body.addEventListener('wheel', _wheelHandler, { passive: false })
```

Comment:
use `on` and `off` in `@element-plus/utils/dom` for attaching/dettaching events. :bow:

Reply:
Also, would you mind adding tests against these changes? If you have trouble doing that I can guide you through it.

Reply:
Plus: please add this to co-responding file `el-image-viewer`, you can add it into the `onMounted` and `onBeforeUnmounted` hooks.

Reply:
ok, i will change code to use `on` ( `off` ) and learn to add tests.

Reply:
@JeremyWuuuuu i try to use `on` function but it's fourth parameter is `useCapture: boolean`, it can't be set as `{passive: false }

Reply:
You can use `addEventListener` first, and I will refactor it by using vueuse later

/cc @JeremyWuuuuu